test/support/assert_api_response.rb

```diff
@@ -55,4 +118,10 @@ def assert_api_request(method, path, params = {})
       end
     end
   end
+
+  def to_auto_paginate(path)
```

Comment:
@rubemz it's documented above on L103. I'll end up moving the docs inline soon. I have some other changes for tests I'm going to implement that involve splitting the pagination helpers into a separate file.